index.ts

```diff
@@ -38,10 +38,7 @@ module.exports = (sails: Sails.Sails): Sails.Hook<SwaggerGenerator> => {
 
       // https://github.com/balderdashy/sails/blob/master/lib/EVENTS.md#routerbind
       sails.on('router:bind', routeObj => {
-        if (!routes.find(route => route.path === routeObj.path && route.verb === routeObj.verb.toLowerCase())) {
-          routeObj.verb = routeObj.verb.toLowerCase();
-          routes.push(routeObj);
-        }
+        routes.push(routeObj);
```

Comment:
can we force the verbs to be `lowercase` before pushing to `routes`:

```ts
 routeObj.verb = routeObj.verb.toLowerCase();
```

see #65

Reply:
Will do; perhaps it is safest to do so.

I had worked on the basis that Sails did the lowercase during its bind operation.

See https://github.com/balderdashy/sails/blob/master/lib/router/bind.js#L469

Reply:
Done https://github.com/theoomoregbee/sails-hook-swagger-generator/pull/67/commits/aea5de5d65253391a6cd55d06d62fdad47c64a7c